src/field/captcha/captcha_pane.jsx

```diff
@@ -7,7 +7,7 @@ import * as l from '../../core/index';
 import { swap, updateEntity } from '../../store/index';
 import * as captchaField from '../captcha';
 import { getFieldValue, isFieldVisiblyInvalid } from '../index';
-import { ReCAPTCHA } from './recaptchav2';
+import { ReCAPTCHA } from './recaptchav2.jsx';
```

Comment:
Why is it necessary to include the extension here? We don't appear to need to do this in other places.